ideas/99-confidence.md

```diff
@@ -90,30 +90,38 @@ Request messages history after background: https://github.com/status-im/status-r
 New protocol https://github.com/status-im/status-react/pull/3273
 https://github.com/status-im/ideas/issues/87
 
-### Minimum Viable Product
-Goal Date: <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+### Security and Privacy Implications
 
-Description: <!-- Description of Deliverables-->
+None currently known.
 
-**TBD**
-Probably something like: Select subset of above to implement/finish; do initial community check, do design basics and plan initial UXR session.
+### Iteration 1
 
-## Dates
-Goal Date: <!-- Date for evaluation in ISO 8601 (yyyy-mm-dd) format --> 
+Complete several reliability improvements that have been identified so far:
 
-Description: <!-- Description of Deliverables-->
+- [#3827 Message reliability survey](https://github.com/status-im/status-react/issues/3827)
+- [#3793 Improve timestamps in chat messages](https://github.com/status-im/status-react/issues/3793)
+- [#3787 Improve network offline and mail server error messaging](https://github.com/status-im/status-react/issues/3787)
+- [#3784 Provide users with delivery status feedback when sending messages](https://github.com/status-im/status-react/issues/3784)
+- [#3792 Measure message send/receive ratio on internal builds ](https://github.com/status-im/status-react/issues/3792)
+- [#828 Send an expiration signal when envelope wasn't delivered to any peer](https://github.com/status-im/status-go/pull/828)
+- [#810 Notify clj side when the message actually "left" local node.](https://github.com/status-im/status-go/issues/810)
+- [#3785 Remove "seen by everyone" from public chat](https://github.com/status-im/status-react/issues/3785)
 
-Testing Days required: <!-- Days required at the end of development for testing -->
+### Iteration 2
+
+After iteration 1 is complete the swarm will meet and discuss the results of the UXR survey, send/receive ratio results, review current UX and discuss future iterations. Work will then be planned for a future iteration, or the swarm will be closed.
+
+## Exit Criteria
+
+- >99% message deliverability from [#3792](https://github.com/status-im/status-react/issues/3792)
```

Comment:
Maybe we should also add max time we want messages to be delivered? For example: "95% messages delivered within 5s for a real-time chat". https://github.com/status-im/status-react/issues/3792 could probably measure it

Reply:
There was a discussion about the timeout, so that could be our cut off?

Reply:
@lukaszfryc  unfortunately is not under our control, as that includes network delays. We can add though the time it takes to dispatch the message and how long it takes after receiving a message to display it to the user.